src/data/script_03_clean_data.py

```diff
@@ -0,0 +1,27 @@
+import pandas as pd
+import os
+import os.path
+
+def clean_file(file_path: str) -> pd.DataFrame:
+    """ Cleans a given CSV file from NaN values etc. and returns the data frame. """
+    print('Loading file ' + file_path)
+    if os.path.isfile(file_path):
+        df = pd.read_csv(file_path, encoding = "latin-1")
+        print('1. Removing empty rows and columns')
+        df_without_na_rows = df.dropna(how = "all")
+        df_without_na_rows_and_cols = df_without_na_rows.dropna(axis = 1, how = "all")
+        return df_without_na_rows_and_cols
+    else:
+        print('The data file does not exist!')
+
+
+def clean_order_data() -> pd.DataFrame:
+    """ Reads and cleans the order data. """
+    return clean_file(r'data/interim/orders/orders_with_headers.csv')
+
+def clean_clickstream_data() -> pd.DataFrame:
+    """ Reads and cleans the clickstream data. """
+    return clean_file(r'data/interim/clickstream/clickstream_with_headers.csv')
+
```

Comment:
Fehlen hier nicht noch zwei Zeilen, in etwa "odf = clean_order_data()" etc?

Reply:
Außerdem sollten wir (fällt mir gerade noch ein) die Daten in R und Python in zwei (bzw. vier) verschiedene Output-Dateien speichern, also "clickstream_with_headers_R.csv" / "clickstream_with_headers_py.csv" oder so ähnlich. Orders dann analog dazu. Dann können wir die Python und R Dateien vergleichen, im Idealfall sind beide Dateien identisch; so wissen wir ob wir korrekt gearbeitet haben 👍

Reply:
Done!